test/functional/autoupdate/aup-basic.bats

```diff
@@ -59,3 +52,45 @@ test_teardown() {
 	assert_status_is_not "$SWUPD_OK"
 
 }
+
+@test "AUT002: Check auto-update status in different root" {
```

Comment:
aren't you doing basically the same thing as the previous test in this one?

Reply:
So the second one is exclusively for checking the status of auto-update. I created a different test because the previous test was getting too big

Reply:
but you are still doing the same stuff from the first test plus some more stuff, so I don't see how that is going to help with that....

Ideally every test should be atomic, so AUT001 should ideally be split into two tests, one which enables autoupdate (from an initially disabled system) and another one which disables it (from an initially enabled system). The same thing for the autoupdate tests you are writing (for checking the status), there should be two tests, one for checking the status when the system has autoupdate disabled and one when is enabled. Those conditions, the ones I'm referring to when saying "initially" are supposed to be set up in the test_setup() function, so the tests would be split into two or more test files.

Having said that, if you still just want to add that other verification I would just do it in the same test (AUT001), the only thing you are not doing from AUT001 in AUT002 is to check the existence of files:
```
# check for files(should exist and point to /dev/null)
run sudo sh -c "test -c $PATH_PREFIX/etc/systemd/system/swupd-update.service"
assert_status_is "$SWUPD_OK"
run sudo sh -c "test -c $PATH_PREFIX/etc/systemd/system/swupd-update.timer"
assert_status_is "$SWUPD_OK"
```
So you could just add your verifications to this test and maybe change the code above with something like this"
```
assert_file_exists $PATH_PREFIX/etc/systemd/system/swupd-update.service
assert_file_exists $PATH_PREFIX/etc/systemd/system/swupd-update.timer
```
At least that way you are saving a couple of lines of code, plus you would be using the assertions from the testlib which is supposed to be the correct way of verifying results.

Reply:
Makes sense. Okay I will add the verifications to the same test.

for the assert_file_exists i remember you asking of this change before..The problem these files are charcters files as they point to /dev/null and assert_file_exists. I will change the assert_file_exist in that case.

Reply:
> for the assert_file_exists i remember you asking of this change before..The problem these files are charcters files as they point to /dev/null and assert_file_exists. I will change the assert_file_exist in that case.

Ahh I didn't remember that, but it makes sense that I complaint about that before haha 😝